### PR TITLE
GTFS - read stops file

### PR DESCRIPTION
**Pull request by prhod**
There is still work on equipments, but that can be done later.

### CODE REVIEW

File: src/gtfs.rs [hunk 1]
```diff
@@ -53,12 +53,89 @@ impl From<Agency> for objects::Company {
...
+    #[serde(rename = "zone_id")]
+    fare_zone_id: Option<String>,
+    #[serde(rename = "stop_url")]
+    url: Option<String>,
+    #[serde(default)]
+    location_type: i32,
+    parent_station: Option<String>,
+    #[serde(rename = "stop_timezone")]
+    timezone: Option<String>,
+    #[serde(default, rename = "wheelchair_boarding")]
```
Comment (datanel) [inline src/gtfs.rs]: do you really need to rename `wheelchair_boarding` to `wheelchair_boarding`?

Comment (prhod) [reply src/gtfs.rs]: you're right ! fixed

File: src/gtfs.rs [hunk 2]
```diff
@@ -53,12 +53,89 @@ impl From<Agency> for objects::Company {
...
+            visible: true,
+            geometry_id: None,
+            equipment_id: None,
+        }
+    }
+}
+impl From<Stop> for objects::StopPoint {
+    fn from(stop: Stop) -> objects::StopPoint {
+        let id = stop.id;
+        let stop_area_id = stop.parent_station.unwrap_or_else(|| format!("SA{}", id));
```
Comment (TeXitoi) [inline src/gtfs.rs]: Also here.

Comment (TeXitoi) [reply src/gtfs.rs]: With it's test.

**Pull request merged**